CONTRIBUTING.md

```diff
@@ -158,12 +170,12 @@ member. The review process is outlined in the [Review guide](REVIEWING.md).
 #### Merge Style
 
 All pull requests are squashed and merged. We generally discourage large pull
-requests that are over 300-500 lines of diff. If you would like to propose
-a change that is larger we suggest coming onto our gitter channel and
-discuss it with one of our engineers. This way we can talk through the
-solution and discuss if a change that large is even needed! This overall
-will produce a quicker response to the change and likely produce code that
-aligns better with our process.
+requests that are over 300-500 lines of diff. If you would like to propose a
+change that is larger we suggest coming onto our gitter channel and discuss it
```

Comment:
Should we change this to point to discord? /cc @zsherman

Reply:
Once we make the full switch I'll add a separate PR I think